src/guide/hotspot-development.md

```diff
@@ -10,27 +10,76 @@
 
 While developing your fix, your might want your code to output some diagnostic information. You might even want to leave some logging in the code you check in, to facilitate future diagnostics.
 
-The appropriate way to print logging output from HotSpot is through the [Unified Logging Framework (JEP 158)](https://openjdk.org/jeps/158). It gives you a lot of nice features and enables common command-line options for all logging. Messages can also be "decorated" with e.g. uptime, level, tags. The JEP contains a thorough description of the feature, but a quick example might look like:
+The appropriate way to print logging output from HotSpot is through the [Unified Logging Framework (JEP 158)](https://openjdk.org/jeps/158).
+It gives you a lot of nice features and enables common command-line options for all logging.
```

Comment:
Please don't add line breaks in sections. Goes for sections below as well.

Reply:
Starting each sentence on a new line seems a very personal style choice - ref:
https://groups.google.com/g/pandoc-discuss/c/tEWDWL2mLBY?pli=1
It probably helps if also trying to maintain maximum line lengths, as editing may need fewer lines to be rewrapped. But personally I find it jarring to read.

Reply:
Okay, this is probably a consequence of my editor being set up for coding and not for text editing then :-).

Reply:
It's a personal style choice and it's the coding style used in the Developers' Guide. If lines are getting too long I recommend making your editor window narrower.